source/llassetgen/source/Image.cpp

```diff
@@ -234,8 +234,7 @@ namespace llassetgen {
                 memcpy(&data[(min.y + y) * stride + min.x / 8],
                        &ft_bitmap.buffer[y * ft_bitmap.pitch],
                        rowLength);
-            }
-        } else {
+            }        } else {
```

Comment:
I'd argue its hard to track nesting here.

Reply:
same as above. sometimes, when I merge files, newlines get deleted.